sormas-backend/src/main/java/de/symeda/sormas/backend/report/WeeklyReportService.java

```diff
@@ -209,8 +208,9 @@ public Predicate createUserFilter(CriteriaBuilder cb, CriteriaQuery cq, From<Wee
 			return usersStream;
 		}
 
+		final JurisdictionLevel jurisdictionLevel = UserRole.getJurisdictionLevel(user.getUserRoles());
```

Comment:
Maybe we should add a transient getJurisdictionLevel method to the user that wraps this and could also cache it? What do you think?

Reply:
ok, good idea